catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -70,17 +73,31 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   }
 
   private <T> void handleValue(Metacard metacard, NitfAttribute attribute, T segment) {
-
     Function<T, Serializable> accessor = attribute.getAccessorFunction();
-    Serializable value = accessor.apply(segment);
+
+    Serializable value;
+    try {
+      value = accessor.apply(segment);
+    } catch (NitfParsingException e) {
+      LOGGER.debug(
```

Comment:
❓ Not sure what the rule is for the ingest log file, but should we log something there when this happens?